src/types/pubsub/PubSubBulkPublishResponse.type.ts

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { PubSubBulkPublishEntry } from "./PubSubBulkPublishEntry.type";
+
+/**
+ * PubSubBulkPublishResponse defines the response from a bulk publish request.
+ */
+export type PubSubBulkPublishResponse = {
+  failedEntries: {
```

Comment:
Split up in separate type please (e.g., PubSubBulkPublishResponseFailedEntry and use it as an array)